src/services/qrService.ts

```diff
@@ -51,6 +51,9 @@ import { getContacts } from './contactService'
 import { getOrCreatePrimaryIdentifier } from './identityService'
 import { NativeStackNavigationProp } from '@react-navigation/native-stack'
 import { translateCorrelationIdToName } from '../utils/CredentialUtils'
+import {
+  PresentationDefinitionWithLocation
+} from '@sphereon/did-auth-siop/dist/main/authorization-response'
```

Comment:
Isn't this exported from the main package? Looking at the code it should be

Reply:
ye it imports by default like this.

changed